### PR TITLE
Update the routing for EstablishClaims

### PR DESCRIPTION
**Pull request by mdbenjam**
Updated the routes, roles, and tests for our new EstablishClaim page.

CC: @joofsh 

**Test Plan**
- Manually smoke test
- Run test suite

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -47,11 +47,11 @@ def index_template
 
   def manager?
     # TODO(jd): Determine real CSS role to be used
-    current_user.can?("manage #{department}")
+    current_user.can?(User::TASK_TYPE_TO_ROLES[type][:manager])
```
Comment (joofsh) [inline app/controllers/tasks_controller.rb]: Is there any case where the `User::TASK_TYPE_TO_ROLES[type]` would return nil? If so the `nil[:manager]` would blow up

File: app/models/user.rb [hunk 2]
```diff
@@ -5,6 +5,10 @@ class User < ActiveRecord::Base
   attr_accessor :roles
   attr_writer :regional_office
 
+  TASK_TYPE_TO_ROLES = {
+    EstablishClaim: { employee: "Establish Claim", manager: "Manage Claim Establishment" }
```
Comment (joofsh) [inline app/models/user.rb]: 😍

File: config/routes.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@
     get 'cancel', on: :member
   end
 
-  scope "/dispatch", department: :dispatch do
+  scope "/dispatch/establishclaim", task_type: :EstablishClaim do
```
Comment (shanear) [inline config/routes.rb]: i feel like words should be hyphen seperated in urls
`/dispatch/establish-claim`

thoughts?

File: app/controllers/tasks_controller.rb (hunk 1 above)
Comment (shanear) [inline app/controllers/tasks_controller.rb]: true, i'm ok with this because feature tests should test that.

File: config/routes.rb (hunk 3 above)
Comment (mdbenjam) [inline config/routes.rb]: Yeah I think that's good idea.

**Pull request merged**